apps/src/code-studio/initSigninState.js

```diff
@@ -35,10 +36,11 @@ export function getUserSignedInFromCookieAndDom() {
  * Determines signin state and dispatches to the store. Shows a dialog asking
  * the user for their age or to sign in if necessary.
  */
-export default function initSigninState(userType) {
+export default function initSigninState(userType, currentUserId) {
   $(document).ready(() => {
     const store = getStore();
     store.dispatch(setUserSignedIn(getUserSignedInFromCookieAndDom()));
+    store.dispatch(setCurrentUserId(currentUserId));
```

Comment:
don't set currentUserId here. Instead, set it only in the entry points where you need it